sycl/include/CL/sycl/memory_enums.hpp

```diff
@@ -0,0 +1,25 @@
+//==-------------- enums.hpp --- SYCL enums --------------------------------==//
```

Comment:
Please rename to memory_enums.hpp

Reply:
Already done. Please, review latest version

Reply:
Not sure about this, I meant to update this comment line.

Reply:
Done